### DISCUSSION TITLE
Argo scheduling policy issue

### DISCUSSION THREAD

**Discussion by journey-wang** [argoproj/argo-workflows#8863] [Q&A] [answered]
## Summary

When running serveral workflows, e.g. A (A1,A2,A3), B(B1,B2,B3), C(C1,C2,C3), each has several nodes. Found that argo running them 
like: A1, B1,C1, A2, B2, C2 ...

How to change the running sequence to A1, A2, A3, B1, B2, B3...

version: v2.12.13
---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by terrytangyuan** [comment]
You should be able to specify dependencies in a DAG.

**Reply by journey-wang** [reply]
Not for this reason. The issue is related with multiple DAGs, argo didn't process them one DAG after one DAG. But mix all tasks together and process one task after one task.

**Comment by sarabala1979** [comment, marked as answer]
you can use `priority` to sequence the workflow  execution  or  `parallelism` to execute  one  by  one

**Reply by journey-wang** [reply]
Thanks very much, I'll try it
